### ISSUE TITLE
MathML is not spoken by screen readers

### ISSUE THREAD

**Issue by becka11y** [readium/readium-js-viewer#228]
Tested on OS X 10.10 in Safari and the Chrome Extension with VoiceOver - VO speaks blank or text for the MathML blocks.  

Tested on Windows 7 in the Chrome Extension with JAWS - it speaks nothing for the MathML blocks.  

I suspect this is because we are rendering via SVG?

**Comment by danielweck** [comment]
Hello @becka11y 
@pkra says:
> This should be fixable in the upcoming release via a new extension we're building; not ideal but what AT vendors suggested.
https://github.com/readium/readium-js-viewer/issues/262#issuecomment-120830173

**Comment by pkra** [comment]
Ah, didn't see this issue. Thanks for mentioning it.

**Comment by danielweck** [comment]
@pkra Waffle.io is just a "visualizer" for the GitHub repositories / issue trackers :)
You can click on the number in the top-left corner of a Waffle.io card, to open the original GitHub issue / pull request: https://waffle.io/readium/readium-js-viewer/cards/544821477abdfe69dab9a51f

**Comment by danielweck** [comment]
For what's it worth, MathML-Cloud does a pretty good job at creating a intelligible utterances for mathematical equations: http://benetech.org/our-programs/literacy/born-accessible/mathml-cloud/
The server is dead though :( https://mathmlcloud.org

**Comment by pkra** [comment]
> Waffle.io is just a "visualizer" for the GitHub repositories

Ah. Thanks for explaining.

> MathML-Cloud does a pretty good job at creating a intelligible utterances 

Yes, we build the underlying technology which we're integrating into MathJax directly this year.

**Comment by danielweck** [comment]
Ah, good to know @pkra 

Point of reference for the Readium team - MathJax accessibility overview ("meta" issue): https://github.com/mathjax/MathJax/issues/1116

**Comment by danielweck** [comment]
Regarding Braille support, note the comments that @NSoiffer (Neil Soiffer of Design Science) made in this discussion (thanks @larsvoigt for bringing this up!):
https://github.com/mathjax/MathJax/issues/1164#issuecomment-94026213
https://www.dessci.com/en/products/mathplayer/

**Comment by danielweck** [comment]
@becka11y MathJax 2.6 now has better a11y support, you can test from the develop branch:

https://readium.firebaseapp.com/?epub=https%3A%2F%2Fcdn.rawgit.com%2FIDPF%2Fepub-testsuite%2Fmaster%2Fcontent%2F30%2Fepub30-test-0100&goto=%7B%22idref%22%3A%22xhtml-mathml-001%22%7D

Thanks @pkra ! :)

**Comment by danielweck** [comment]
Documentation: http://docs.mathjax.org/en/latest/misc/accessibility-features.html#screenreader-support

**Comment by pkra** [comment]
Since MathMLCloud is mentioned, once we make a first public release for it, I'd love to talk about seeing this in Readium https://github.com/mathjax/MathJax-RespEq/wiki/Support-Matrix-a11y-tool.

**Comment by pkra** [comment]
BTW, the current release works well with Orca+Firefox (ubuntu 15.10).
